### PR TITLE
Update templates to use new and improved context.log

### PR DESCRIPTION
**Pull request by mathewc**
context.log now supports printf like capabilities (i.e. anything Node util.format does). Applying this to the templates and doing some other minor cleanup.

### CODE REVIEW

File: Templates.sln [hunk 1]
```diff
@@ -119,7 +119,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "TimerTrigger", "TimerTrigge
 	ProjectSection(SolutionItems) = preProject
 		Templates\TimerTrigger\function.json = Templates\TimerTrigger\function.json
 		Templates\TimerTrigger\index.js = Templates\TimerTrigger\index.js
-		Templates\TimerTrigger\logger.js = Templates\TimerTrigger\logger.js
```
Comment (mathewc) [inline Templates.sln]: This file is outdated and not referenced in the code.

File: Templates/ServiceBusQueueTrigger/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿module.exports = function (message, context) {
-    context.log("Node.js ServiceBus queue trigger function processed message '" + JSON.stringify(message) + "'");
+    context.log('Node.js ServiceBus queue trigger function processed message', message);
```
Comment (mathewc) [inline Templates/ServiceBusQueueTrigger/index.js]: message will automatically be passed to util.inspect for stringification

File: Templates/BlobTrigger/index.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ﻿module.exports = function (workItem, context) {
-    console.log('Node.js blob trigger function processed work item ' + workItem.id);
+    context.log('Node.js blob trigger function processed work item', workItem.id);
```
Comment (brettsam) [inline Templates/BlobTrigger/index.js]: is this missing a `$s` in the string?

Comment (mathewc) [inline Templates/BlobTrigger/index.js]: Not needed. Any params not matching a % placeholder are just concatenated on with spaces separating :)

Comment (brettsam) [inline Templates/BlobTrigger/index.js]: Oh, nice! I didn't know it supported that.

**Comment by brettsam** [comment]
looks good to me

**Comment by soninaren** [comment]
:+1:

**Comment by soninaren** [comment]
sent a new PR #58 because of merge conflicts

**Pull request closed without merging**